src/main/java/frc/robot/commands/AutoIntakeSequence.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.commands;
```

Comment:
you need to actually register these as NamedCommands in RobotContainer (following last year's code)